distro/run/qa/integration-tests/src/test/java/org/camunda/bpm/run/qa/AutoDeploymentIT.java

```diff
@@ -133,9 +130,9 @@ public void shouldAutoDeployScriptAndForms() throws IOException {
     // when
     Response resourcesResponse = when().get(container.getBaseUrl() + DEPLOYMENT_ENDPOINT + "/" + deploymentId + "/resources");
     List<String> resourceNames = extractResourceNames(resourcesResponse);
-    
+
     // then
-    assertThat(resourceNames).contains("/form.html", "/script.js");
+    assertThat(resourceNames, contains("/form.html", "/script.js"));
```

Comment:
Why this change from assertj to hamcrest?

Reply:
I (only vaguely, to be honest) remember that we had an issue in the EE repo with this test because the assertj dependency wasn't present there somehow. Miklas used it "by accident" and was fine with using hamcrest here because that's on the classpath already. Makes the build easier, I would say. 
We could also investigate on why it didn't work in the EE repo but I think it's not worth the effort for that one assertion right now.
Thought on this?

Reply:
I think we should try to make assertj work. Assertj is our go-to library for assertions. The next person that writes a test here will run into the same problem again and it looks like the root cause is not easy to understand.

Reply:
I'll have a look and update here with insights

Reply:
OK, using assertj again, the EE module now adds a dependency to assertj as well 👍